test/integration/suites/k8s/conf/server/spire-server.yaml

```diff
@@ -147,6 +147,15 @@ data:
       }
     }
 
+    health_checks {
+      listener_enabled = true
+      bind_address = "0.0.0.0"
+      bind_port = "80"
```

Comment:
Can we use a different port other than 80? We're actively exploring ways to run SPIRE as a non-root user in containers and binding to port 80 will require elevated capabilities, so this will end up breaking.

Reply:
I fixed it with https://github.com/spiffe/spire/pull/2015/commits/358825e1da313168c652427dc02f419b17f8466b.

Reply:
Also, I updated the k8s-workload-registrar.conf in suites/k8s-reconcile with https://github.com/spiffe/spire/pull/2015/commits/a79c88c85d288c271ebef0c529f8706f99916396 because the default value of metrics_addr is ":8080", which conflicts with the health check port.